<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thruster</class>
 <widget class="QWidget" name="Thruster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="17" column="1" colspan="4">
     <widget class="QLabel" name="label_SetBackwardK">
      <property name="text">
       <string>Backward K</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QCheckBox" name="CheckBox_ThrusterPower">
      <property name="text">
       <string>Power Thruster</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="6">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="0" colspan="6">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="0" colspan="6">
     <widget class="QLineEdit" name="LineEdit_SetForwardK"/>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="0" colspan="6">
     <widget class="QLineEdit" name="LineEdit_ThrusterSetSpeed"/>
    </item>
    <item row="9" column="0" colspan="6">
     <widget class="QPushButton" name="PushButton_ThrusterSetBackward">
      <property name="text">
       <string>Backward</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0" colspan="6">
     <widget class="QLineEdit" name="LineEdit_SetBackwardSaturation"/>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLabel" name="Label_ThrusterName">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="6">
     <widget class="QPushButton" name="PushButton_ThrusterSetStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="6">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QPushButton" name="PushButton_ThrusterSetForward">
      <property name="text">
       <string>Forward</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="6">
     <widget class="QLineEdit" name="LineEdit_SetForwardSaturation"/>
    </item>
    <item row="24" column="0" colspan="6">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="1" colspan="4">
     <widget class="QLabel" name="label_SetForwardK">
      <property name="text">
       <string>Forward K</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="6">
     <widget class="QLineEdit" name="LineEdit_SetBackwardK"/>
    </item>
    <item row="20" column="1" colspan="4">
     <widget class="QLabel" name="label_SetForwardSaturation">
      <property name="text">
       <string>Forward Saturation</string>
      </property>
     </widget>
    </item>
    <item row="22" column="1" colspan="4">
     <widget class="QLabel" name="label_SetBackwardSaturation">
      <property name="text">
       <string>Backward Saturation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="SpinBox_ThrusterId"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_ThrusterId">
      <property name="text">
       <string>Id</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="2" colspan="2">
     <widget class="QLabel" name="label_SetSpeed">
      <property name="text">
       <string>Speed %</string>
      </property>
     </widget>
    </item>
    <item row="25" column="2" colspan="2">
     <widget class="QCheckBox" name="CheckBox_ThrusterReverse">
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
